pep-0625.rst

```diff
@@ -73,63 +77,87 @@ is available in the file name.
 Specification
 =============
 
-The name of an sdist should be ``{distribution}-{version}.sdist``.
+The name of an sdist should be ``{distribution}-{version}.tar.gz``.
 
 * ``distribution`` is the name of the distribution as defined in :pep:`345`,
-  and normalised according to :pep:`503`, e.g. ``'pip'``, ``'flit-core'``.
+  and normalised as described in `the wheel spec`_ e.g. ``'pip'``,
+  ``'flit_core'``.
 * ``version`` is the version of the distribution as defined in :pep:`440`,
-  e.g. ``20.2``.
+  e.g. ``20.2``, and normalised according to the rules in that PEP.
+
+An sdist must be a gzipped tar archive in pax format, that is able to be
+extracted by the standard library ``tarfile`` module with the open flag
+``'r:gz'``.
 
-Each component is escaped according to the same rules as :pep:`427`.
+Code that produces an sdist file MUST give the file a name that matches this
+specification. In particular the specification of the ``build_sdist`` hook from
+:pep:`517` is extended to require this naming convention.
```

Comment:
Interesting point. I'd taken the view that this PEP defines "the filename of a sdist", so that for a file to be a sdist it must follow that naming convention. The reference to PEP 517 was only meant as a clarification.

> Otherwise "code that produces an sdist" might be too broad, since it can be interpreted to include frontends.

It's *intended* to include frontends, in that sense. Anything that claims to be producing a sdist must name it according to this spec, or they haven't created a sdist. Just the same as with wheels - if you create a zipfile with the right structure, but don't name it correctly, you can't claim you created a wheel.

If you can think of a better wording, I'm happy to accept a suggestion.